react/MuiCozyTheme/theme.js

```diff
@@ -40,6 +43,22 @@ export const theme = createMuiTheme({
     borderRadius: defaultValues.borderRadius
   },
   overrides: {
+    MuiOutlinedInput: {
+      root: {
+        '&$disabled': {
+          background: 'var(--paleGrey)'
+        },
+        '&:hover $notchedOutline': {
+          borderColor: 'var(--coolGrey)' // Does not seem to work, IDK why :/
```

Comment:
should you add a todo to fix it if needed? Or should we remove this config if it's not working and we don't want it?

Reply:
I removed this code from the PR and I will open another one with this code, I may need a pairing on this.